### PR TITLE
[Relay][Pytorch] Add support for `aten::bitwise_and`

### LINKED ISSUE #16005: [PyTorch/NotImplementedError]: The following operators are not implemented: ['aten::bitwise_and']
### Steps to reproduce

```python
import torch
def func(x1, x2):
    return torch.bitwise_and(x1, x2)

x1 = torch.tensor([True, True, False], device="cuda")
x2 = torch.tensor([False, True, False], device="cuda")
func(x1, x2) # --> runs
torch.compile(func, backend="tvm")(x1, x2) # --> not implemented error
```

### Triage

Please refer to the list of label tags [here](https://github.com/apache/tvm/wiki/Issue-Triage-Labels) to find the relevant tags and add them below in a bullet format (example below).

* needs-triage

### PR DESCRIPTION
**Pull request by mshr-h**
Fix #16005
Support [aten::bitwise_and](https://pytorch.org/docs/stable/generated/torch.bitwise_and.html).

cc @jikechao @@Hzfengsy @vvchernov

### CODE REVIEW

**Review by vvchernov — APPROVED** [review]
LGTM

**Review by jikechao — APPROVED** [review]
LGTM, @mshr-h thanks for your PR.

**Review by Hzfengsy — APPROVED** [review]
